backend/mega/mega.go

```diff
@@ -216,7 +218,40 @@ func NewFs(ctx context.Context, name, root string, m configmap.Mapper) (fs.Fs, e
 	defer megaCacheMu.Unlock()
 	srv := megaCache[opt.User]
 	if srv == nil {
-		srv = mega.New().SetClient(fshttp.NewClient(ctx))
+		// srv = mega.New().SetClient(fshttp.NewClient(ctx))
+
+		// Workaround for Mega's use of insecure cipher suites which are no longer supported by default since Go 1.22.
+		// Relevent issues:
+		// https://github.com/rclone/rclone/issues/8565
+		// https://github.com/meganz/webclient/issues/103
+		clt := fshttp.NewClient(ctx)
+		clt.Transport = fshttp.NewTransportCustom(ctx, func(t *http.Transport) {
+			t.TLSClientConfig.CipherSuites = []uint16{
```

Comment:
Rather than hard coding Go's default ciphers, you can read them like this

```go
import "crypto/tls"

func cipherIDs() []uint16 {
    var ids []uint16
    // Read default ciphers
    for _, cs := range tls.CipherSuites() {
        ids = append(ids, cs.ID)
    }
    // Insecure but Mega uses TLS_RSA_WITH_AES_128_GCM_SHA256 for storage endpoints
    // (e.g. https://gfs302n114.userstorage.mega.co.nz) as of June 18, 2025.
    ids = append(ids, tls.TLS_RSA_WITH_AES_128_GCM_SHA256)
    return ids
}

// … later …
t.TLSClientConfig.CipherSuites = cipherIDs()
```